crates/ruff_linter/src/checkers/ast/analyze/statement.rs

```diff
@@ -1474,6 +1474,9 @@ pub(crate) fn statement(stmt: &Stmt, checker: &mut Checker) {
                     checker, stmt, body, handlers, orelse, finalbody,
                 );
             }
+            if checker.enabled(Rule::SuppressedException) {
```

Comment:
The rule name is very close to `SIM105` [`suppressible-exception`](https://docs.astral.sh/ruff/rules/suppressible-exception/). 

Which makes me wonder if the rule should be integrated into `PLW0133`? @AlexWaygood what do you think?

Reply:
I agree the name is similar, but this check is about checking if there is only `add_note` executed on the exception, so it's quite different from SIM105. On the other hand it is similar to PLW0133, but this rule exists in the flake8 bugbear, so it might be worth keeping the same set of rules with upstream..

Reply:
Integrating it into `PLW0133` is appealing because it's a more general rule, and I agree that this is just trying to tackle a variant of that problem.

But flake8-bugbear is more popular and widely used than pylint, I think, and I believe people much more often select our `B` rules than our `PLW` rules. So I agree that consistency with flake8-bugbear is useful here.

I think that when doing rule recategorisation (a medium-term goal that you don't have to worry about here @divident), we'll probably no longer have categories that refer to older linters, and at that point it would probably make sense to merge `PLW0133` and this rule. But for now I guess I'd go for a separate check, despite how similar the two rules are.

Reply:
It feels strange to me to introduce a new rule if we know we'll want to merge it eventually anyway. But I do see the point and redirects are relatively cheap for users.

Reply:
But I think we have to come up a more specific name to also make it clear that this is a very specific rule and it doesn't aim to catch all suppressed exceptions.

Reply:
What about the name "add_note_exception_suppression" ? 

A few more suggestions:
1. add_note_only_exception
2. note_only_exception_suppression
3. exclusive_add_note_exception
4. exception_note_only
5. suppressed_with_add_note